usb_comm:
  interface_usb_vid: 0x0483
  interface_usb_pid: 0x5740
  interface_usb_read_endpoint: 0x81
  interface_usb_write_endpoint: 0x01
  interface_usb_read_timeout: 1
  interface_usb_write_timeout: 1